### PR TITLE
Feature/vz 480 Add deployment tools into Venzee module

### PR DESCRIPTION
**Pull request by gartox**

### CODE REVIEW

File: tools/clearcdncache [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+
+host=$1
+dns_provider_url=$2
```
Comment (Menshawi) [inline tools/clearcdncache]: no need, this will only work with Cloudflare

File: tools/deployfeature.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+echo "Ensuring website settings for bucket \"${s3_bucket}\""
+aws s3 website $s3_bucket --index-document index.html --error-document index.html
+
+echo "Syncing build with \"${s3_bucket}\""
+
+aws s3 sync --delete --acl public-read /$PROJECT_BASE_BUILD_PATH/$PROJECT_TO_DEPLOY_PATH $s3_bucket >> /build/sync.log
+
+echo
+dns_record_name=${feature_id}-$ENV.venzee.com
```
Comment (Menshawi) [inline tools/deployfeature.sh:21]: do you need to change this?

**Review by Menshawi — CHANGES_REQUESTED** [review]

File: tools/clearcdncache (hunk 1 above)
Comment (gartox) [reply tools/clearcdncache]: I should remove this file?

Comment (gartox) [reply tools/clearcdncache]: I should remove this file?

Comment (Menshawi) [reply tools/clearcdncache]: not the whole file, but this parameter, dns_provider_url should be a constant in this file

Comment (Menshawi) [reply tools/clearcdncache]: You shouldn't need to pass it from the caller of this script

Comment (gartox) [reply tools/clearcdncache]: You mean should be 
`dns_provider_url="https://api.cloudflare.com/client/v4/zones/${CF_ZONE}"`

???

**Review by Menshawi — APPROVED** [review]

**Pull request merged**